Source/santad/DataLayer/WatchItemPolicy.h

```diff
@@ -29,32 +28,63 @@ enum class WatchItemPathType {
   kLiteral,
 };
 
+static constexpr WatchItemPathType kWatchItemPolicyDefaultPathType =
+    WatchItemPathType::kLiteral;
+static constexpr bool kWatchItemPolicyDefaultAllowReadAccess = false;
+static constexpr bool kWatchItemPolicyDefaultAuditOnly = true;
+
 struct WatchItemPolicy {
-  WatchItemPolicy(std::string_view n, std::string_view p, bool wo = false,
-                  WatchItemPathType pt = WatchItemPathType::kLiteral,
-                  bool ao = true, std::set<std::string> &&abp = {},
-                  std::set<std::string> &&ati = {},
-                  std::set<std::array<uint8_t, CS_CDHASH_LEN>> &&ach = {},
-                  std::set<std::string> &&acs = {})
+  struct Process {
+    Process(std::string bp, std::string ti, std::vector<uint8_t> cdh,
+            std::string ch)
+        : binary_path(bp),
+          team_id(ti),
+          cdhash(std::move(cdh)),
+          certificate_sha256(ch) {}
+
+    bool operator==(const Process &other) const {
+      return binary_path == other.binary_path && team_id == other.team_id &&
```

Comment:
It'd be nice to keep one condition per-line, it's more readable.

Of course, if clang-format overrides this, then ignore.

Reply:
clang-format won't let that slide :(